app/views/orders/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<br>
+<div class="d-flex justify-content-evenly">
+  <%= render "dropdown" %>
+  <%= render "search" %>
+</div>
+<h1 class="text-center">Orders</h1>
+
+<div class="w-50 text-end mx-auto p-2">
+  <%= link_to "New Order", new_order_path, class: "text-decoration-none text-light btn btn-info" %>
+</div>
+
+<table class="table w-50 table-bordered table-striped table-hover m-auto text-center">
+  <tr>
+    <th>Index</th>
+    <th>Product Name</th>
```

Comment:
We have used "Product" as "Commodity" so, Can we please replace it with "Commodity Name"